extensions/email/src/main/java/org/mule/extension/email/internal/retriever/RetrieverOperations.java

```diff
@@ -95,20 +99,22 @@ public void store(@Connection RetrieverConnection connection,
      * Eliminates from the mailbox the email with id {@code emailId}, if no {@code emailId} is specified will look for
      * incoming emails in the {@link MuleMessage} it could be a single or multiple emails.
      * <p>
-     * For IMAP mailboxes all the messages scheduled for deletion (marked as DELETED) will also be erased from the folder if the operation succeed.
+     * For IMAP mailboxes all the messages scheduled for deletion (marked as DELETED) will also be erased from the folder
+     * if the operation succeed.
      * <p>
-     * If no {@code emailId} is provided and no emails are found in the incoming {@link MuleMessage} this operation will fail
-     * and no email is going to be erased from the folder, not even the ones marked as DELETED previously.
+     * If no {@code emailId} is provided and no emails are found in the incoming {@link MuleMessage} this operation will
+     * fail and no email is going to be erased from the folder, not even the ones marked as DELETED previously.
      *
      * @param message       the incoming {@link MuleMessage}.
      * @param connection    the corresponding {@link RetrieverConnection} instance.
-     * @param mailboxFolder the folder where the emails are going to be fetched
+     * @param mailboxFolder the folder where the emails are going to be deleted
      * @param emailId       an optional email number to look up in the folder, if there is no email in the incoming {@link MuleMessage}.
      */
+    @Summary("Deletes an email from the given Mailbox Folder")
     public void delete(MuleMessage message,
                        @Connection RetrieverConnection connection,
-                       @Optional(defaultValue = INBOX_FOLDER) String mailboxFolder,
-                       @Optional Integer emailId)
+                       @Optional(defaultValue = INBOX_FOLDER) @Summary("Mailbox folder where the emails are going to be deleted") String mailboxFolder,
+                       @Optional @Summary("Email ID Number of the email to deleted. If not provided it will be taken from the incoming Mule Message") @DisplayName("Email ID") Integer emailId)
```

Comment:
this summary things is making the code unreadable and are not giving information that the javadoc shouldn't be giving.